src/stackdriver-nozzle/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"github.com/evandbrown/gcp-tools-boshrelease/src/stackdriver-nozzle/firehose"
+	"github.com/evandbrown/gcp-tools-boshrelease/src/stackdriver-nozzle/stackdriver"
+
+	"github.com/evandbrown/gcp-tools-boshrelease/src/stackdriver-nozzle/dev"
+	"github.com/evandbrown/gcp-tools-boshrelease/src/stackdriver-nozzle/nozzle"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	debug = kingpin.Flag("debug", "send events to stdout").
+		Default("false").
+		OverrideDefaultFromEnvar("DEBUG").
+		Bool()
+	apiEndpoint = kingpin.Flag("api-endpoint",
+		"CF API endpoint (use https://api.bosh-lite.com for BOSH Lite)").
+		OverrideDefaultFromEnvar("API_ENDPOINT").
+		Required().
+		String()
+	username = kingpin.Flag("username", "username").
+			Default("admin").
+			OverrideDefaultFromEnvar("FIREHOSE_USERNAME").
+			String()
+	password = kingpin.Flag("password", "password").
+			Default("admin").
+			OverrideDefaultFromEnvar("FIREHOSE_PASSWORD").
+			String()
+	skipSSLValidation = kingpin.Flag("skip-ssl-validation", "please don't").
+				Default("false").
+				OverrideDefaultFromEnvar("SKIP_SSL_VALIDATION").
+				Bool()
+	projectID = kingpin.Flag("project-id", "gcp project id").
+			OverrideDefaultFromEnvar("PROJECT_ID").
+			String() //maybe we can get this from gcp env...? research
+)
+
+func main() {
+	//todo: pull in logging library...
```

Comment:
We have straight up print statements in here right now, but we really should use a real logging library so we can filter them by debug/info/etc...